### PR TITLE
Pensions pow severance

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#2223

Matches UI to back end schema

screen shot 2017-06-09 at 1 25 35 pm

### CODE REVIEW

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -199,10 +199,28 @@ const formConfig = {
...
               'ui:options': {
                 expandUnder: 'view:receivedSeverancePay'
+              },
+              amount: {
+                'ui:title': 'Amount'
+              },
+              type: {
+                'ui:title': 'Pay Type',
+                'ui:widget': 'radio',
+                'ui:options': {
```
Comment (jbalboni) [inline src/js/pensions/config/form.js]: You probably don't need the labels here if they're the same as the enum values.

**Review by jbalboni — APPROVED** [review]
LGTM, one minor comment.

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -199,10 +199,28 @@ const formConfig = {
               'ui:widget': 'yesNo'
             },
             severancePay: {
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Thanks @J-Quag, @jbalboni for noting the UI for this needed to be synced up with the schema.

File: src/js/pensions/config/form.js (hunk 1 above)
Comment (cehsu) [inline src/js/pensions/config/form.js]: Thanks @jbalboni, I thought so too initially, but removing these causes a `enumOptions.map is not a function` error.

**Comment by J-Quag** [comment]
Looks good :)

**Pull request merged**